blockchain/blockchain.go

```diff
@@ -481,17 +470,37 @@ func (b *BlockChain) CreateCRCAppropriationTransaction() (*Transaction, error) {
 		return nil, nil
 	}
 	outputs := []*OutputInfo{{b.chainParams.CRCCommitteeAddress,
-		&appropriationAmount}}
+		appropriationAmount}}
 
 	var tx *Transaction
-	tx, err = b.createTransaction(&payload.CRCAppropriation{},
+	tx, err = b.createTransaction(&payload.CRCAppropriation{}, CRCAppropriation,
 		b.chainParams.CRCFoundation, Fixed64(0), uint32(0), utxos, outputs...)
 	if err != nil {
 		return nil, err
 	}
 	return tx, nil
 }
 
+func (b *BlockChain) CreateCRRealWithdrawTransaction(
+	withdrawTransactionHashes []Uint256, outputs []*OutputInfo) (*Transaction, error) {
+	utxos, err := b.getUTXOsFromAddress(b.chainParams.CRCCommitteeAddress)
+	if err != nil {
+		return nil, err
+	}
+
+	payload := &payload.CRCProposalRealWithdraw{
+		WithdrawTransactionHashes: withdrawTransactionHashes,
+	}
+
+	var tx *Transaction
+	tx, err = b.createTransaction(payload, CRCProposalRealWithdraw,
+		b.chainParams.CRCFoundation, Fixed64(RectifyTxFee), uint32(0), utxos, outputs...)
```

Comment:
using RealWithdrawSingleFee instead of RectifyTxFee

Reply:
fixed